hedera-node/src/main/java/com/hedera/services/records/TxnAwareRecordsHistorian.java

```diff
@@ -54,16 +53,19 @@
 	private final ExpiryManager expiries;
 	private final TransactionContext txnCtx;
 	private final Supplier<FCMap<MerkleEntityId, MerkleAccount>> accounts;
+	private final Supplier<FCMap<MerkleEntityId, MerkleSchedule>> schedules;
 
 	public TxnAwareRecordsHistorian(
 			RecordCache recordCache,
 			TransactionContext txnCtx,
 			Supplier<FCMap<MerkleEntityId, MerkleAccount>> accounts,
+			Supplier<FCMap<MerkleEntityId, MerkleSchedule>> schedules,
```

Comment:
This to me is another hint that the `RecordsHistorian` isn't the right component to centralize entity expiration...the responsibility of this component is supposed to be just managing records, so it shouldn't need the `schedules` map.